libraries/AP_HAL_Linux/Scheduler.cpp

```diff
@@ -60,6 +60,8 @@ extern const AP_HAL::HAL& hal;
         .rate = APM_LINUX_##UPPER_NAME_##_RATE,                 \
     }
 
+#define DEBUG_STACK 0
+
```

Comment:
See the PR message :)

I don't want to add yet a more param for that, particularly because it's meant only for developers and a very small portion of the developers. I'll take a look on a better build sys integration.